modules/c++/math.linear/include/math/linear/Line2D.h

```diff
@@ -0,0 +1,68 @@
+/* =========================================================================
+ * This file is part of math.linear-c++
+ * =========================================================================
+ *
+ * (C) Copyright 2004 - 2014, MDA Information Systems LLC
+ *
+ * math.linear-c++ is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this program; If not,
+ * see <http://www.gnu.org/licenses/>.
+ *
+ */
+#ifndef __MATH_LINEAR_LINE_2D_H__
+#define __MATH_LINEAR_LINE_2D_H__
+
+#include <types/RowCol.h>
+#include <except/Exception.h>
+#include <sys/Conf.h>
+#include <iostream>
+#include <cmath>
+
```

Comment:
Only include headers needed, and only where they're needed... so here, Exception.h, Conf.h, and cmath are only needed in the cpp I think.  I don't think iostream is needed at all is it?  Also, put system headers before our headers.